tasks/generate_templates.yml

```diff
@@ -16,6 +16,40 @@
     - encryption-config.yml
     - ssh-key.pub
 
+- name: "({{ cluster.name }}) verify kops_dockerconfig_creds for dockerhub"
+  uri:
+    url: "{{ kops_dockerconfig_creds.url | default(kops_default_dockerconfig.url )}}"
```

Comment:
I don't think we need a default here as this piece of code is never reached. If `kops_dockerconfig_creds` is not defined, this block will not be executed.

Reply:
We still need `url` in default until we want to access private registry. Then we don't define `url` in `kops_dockerconfig_creds` in vault when we try to login dockerhub.